test/unit/fh3/test_preview.js

```diff
@@ -0,0 +1,35 @@
+var assert = require('assert');
+var genericCommand = require('genericCommand');
+var _ = require('underscore');
+var showPreviewCmd = genericCommand(require('cmd/fh3/preview/show'));
+var urlPreviewCmd = genericCommand(require('cmd/fh3/preview/url'));
+
+var nock = require('nock');
+var data = {
+  "url": "http://henr.ie/2s7DFXD",
+  "longUrl": "https://support-cuke6ultphkoxaulyhpsb6ep-dev.mbaas1.us.feedhenry.com"
+};
+
+var data = "https://support.us.feedhenry.com/box/srv/1.1/wid/support/studio/1a/container";
+
+module.exports = nock('https://apps.feedhenry.com')
+  .get('box/srv/1.1/wid/', '*')
+  .times(2)
+  .reply(200, data);
+
+module.exports = {
+  'test fhc preview show --app': function(cb) {
+    urlPreviewCmd({app:'1a'}, function(err, data) {
+      assert.ok(!err, "No Error is Expected");
+      assert.ok(data, "Data is Expected");
+      return cb();
+    });
+  },
```

Comment:
are these not the same test? I think it would be good to assert that the url that comes back is correct also